fdm-core/src/db/schema-custom-types.ts

```
@@ -216,27 +225,54 @@ function readPolygon(
-//         offset += polygonSize
-//     }
+function readMultiPolygon(
+    dataView: DataView,
+    littleEndian: boolean,
+    offset: number,
+): GeoJSON.Position[][][] {
+    if (offset + 4 > dataView.byteLength) {
+        throw new Error("Buffer too small to read MultiPolygon")
+    }
+    const numPolygons = dataView.getUint32(offset, littleEndian)
+    offset += 4
```

Biome (2.4.16)

[error] Assigning a function parameter is confusing.

(lint/style/noParameterAssign)